### PR TITLE
Update metronome extension

### PR DESCRIPTION
**Pull request by Visual-Studio-Coder**
## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

**Comment by pernielsentikaer** [comment]
Seems like you are pushing the complete extension again, could you look into only pushing the new changes @Visual-Studio-Coder

**Comment by Visual-Studio-Coder** [comment]
@pernielsentikaer i actually HAVE made some changes. I mean, the change I made IS pretty small because I was just fixing a typo I had noticed

**Comment by Visual-Studio-Coder** [comment]
How do I fix the merge conflicts?

**Comment by pernielsentikaer** [comment]
Maybe if it's just a small correction you delete the local version of the extension and fork it again

**Comment by Visual-Studio-Coder** [comment]
Alright, I fixed it.

File: extensions/metronome/src/index.tsx [hunk 1]
```diff
@@ -62,12 +60,13 @@ export default function Command(props: LaunchProps) {
...
     Number.isInteger(Number(group)) &&
     Number(group) > 0 &&
-    Number(group) < 700
+    Number(group) < 500
   ) {
-    const description = isRunning ? "Click ↵ to stop the timer." : "Click ↵ to start the timer.";
+    const description = isRunning ? "Click ↵ to pause" : "Click ↵ to play";
+
     return (
       <List searchBarPlaceholder="" searchText="">
```
Comment (pernielsentikaer) [inline extensions/metronome/src/index.tsx:71]:
```suggestion
      <List searchBarPlaceholder="">
```

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi 👋 

Thanks for your contribution 💪 

I have now tested your extension, and I have some feedback ready for you:

- Make `Accents` optional and let the user add a preference for the default (could be 4)

I'm looking forward to testing this extension again 🔥 

Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.

**Comment by pernielsentikaer** [comment]
What do you think @Visual-Studio-Coder

**Comment by Visual-Studio-Coder** [comment]
@pernielsentikaer I finished it after a long time!

**Pull request merged**